Assets/Scripts/Inventory/Inventory.cs

```diff
@@ -82,7 +88,7 @@ public void CmdMoveItem(GameObject fromContainer, int fromIndex, GameObject toCo
         var to = toContainer.GetComponent<Container>();
 
         if (!Container.AreCompatible(to.GetSlot(toIndex), from.GetItem(fromIndex).itemType))
-            throw new System.Exception("Item not compatible with slot");
+            throw new InventoryOperationException("Item not compatible with slot");
```

Comment:
I would rather that we don't use exceptions. They break the flow too much

Reply:
The use of exceptions in general should probably be a separate discussion.
I think this code does use exceptions in the way they are intended, as an exception only occurs if the code does not check whether or not the slot it's trying to move an item to is compatible (which is an exceptional circumstance).

Reply:
Discussion is continued in Discord